hack/azure-jump-host/jump-host-create

```diff
@@ -5,6 +5,7 @@ set -o pipefail
 SCRIPTDIR="$( dirname --  $(realpath "${BASH_SOURCE}"); )";
 RG=$(jq -r .azureresourcegroup constellation-state.json)
 SUBNET=$(jq -r .azuresubnet constellation-state.json)
+LOCATION=$(jq -r .azurelocation constellation-state.json)
```

Comment:
What version have you tested this with? We don't have a state file on the current main branch?

Reply:
I needed this to recover a v2.0 cluster. Maybe this should be on the v2.0 branch? The docs for v2.0 in the recovery section could also point to this hack folder. They currently only state that you should setup a jump host.

Reply:
I think that's a good idea. Can you rebase this PR to target `release/v2.0`?